### PR TITLE
Initial start on dashboarding

### PR DESCRIPTION
**Pull request by stokkie90**

### CODE REVIEW

**Comment by rick2909** [comment]
just a tip if you dont want the pull request not be merged. Use the Draft feature from github
[Converting a pull request to a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft)

**Comment by CM000n** [comment]
> @rick2909 Thanks! used to gitlab 😅

It‘s the same in gitlab 😉

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -20,6 +20,11 @@ repos:
       - id: "check-merge-conflict"
       - id: "check-yaml"
         args: ["--unsafe"]
+        exclude: |
+            (?x)^(
+                custom_components/ui_lovelace_minimalist/installation/.*\.yaml|
```
Comment (CM000n) [inline .pre-commit-config.yaml:25]: Why do you add these folders/data to be ignored by "chack-yaml"?

**Comment by stokkie90** [comment]
Not entirely though. Gitlab works if you add WIP or DRAFT  in the title. 

But indeed not possible to merge works the same. Just didn't know GitHub had this feature.

File: .yamllint [hunk 2]
```diff
@@ -32,7 +32,7 @@ rules:
   new-lines: "enable"
   octal-values: "disable"
   quoted-strings:
-    quote-type: "double"
+    quote-type: "any"
```
Comment (CM000n) [inline .yamllint:35]: I am definitely against this adjustment. Our .yaml files follow the Home Assistant Style Guide for .yaml files, which prescribes double-quoted strings. I see no reason why we should not apply this to the rest of the .yaml files in the repo as well

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -33,7 +38,13 @@ repos:
     hooks:
       - id: "yamllint"
         args: ["--format", "parsable", "--strict"]
-        exclude: ".pre-commit-config.yaml"
+        # exclude: ".pre-commit-config.yaml"
+        exclude: |
+            (?x)^(
+                .pre-commit-config.yaml|
+                custom_components/ui_lovelace_minimalist/installation/.*\.yaml|
```
Comment (CM000n) [inline .pre-commit-config.yaml:45]: Why do you add these folders/data to be ignored by "yamllint"?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (stokkie90) [reply .pre-commit-config.yaml:25]: Because of Jinja templating and this check not liking it. 😅

File: .yamllint (hunk 2 above)
Comment (stokkie90) [reply .yamllint:35]: Check, will reverse it.

File: .pre-commit-config.yaml (hunk 3 above)
Comment (stokkie90) [reply .pre-commit-config.yaml:45]: Jinja templating.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (stokkie90) [reply .pre-commit-config.yaml:25]: Might think of changing it to extension based. But requires some more work.

Comment (CM000n) [reply .pre-commit-config.yaml:25]: Ahh, good to know.
What's exactly the problem with jinja templating?
What error is it throwing?

Comment (stokkie90) [reply .pre-commit-config.yaml:25]:
```
while scanning for the next token
found character that cannot start any token
  in "custom_components/ui_lovelace_minimalist/lovelace/views/main/rooms/room.yaml", line 5, column 2
while scanning for the next token
found character that cannot start any token
  in "custom_components/ui_lovelace_minimalist/lovelace/views/main/01_home.yaml", line 8, column 6
````

**Pull request closed without merging**